client/src/components/categorical/category/index.tsx

```diff
@@ -286,22 +277,17 @@ class Category extends React.PureComponent {
 
   render(): JSX.Element {
     const {
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'metadataField' does not exist on type 'R... Remove this comment to see the full error message
       metadataField,
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'isExpanded' does not exist on type 'Read... Remove this comment to see the full error message
       isExpanded,
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'categoricalSelection' does not exist on ... Remove this comment to see the full error message
       categoricalSelection,
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'crossfilter' does not exist on type 'Rea... Remove this comment to see the full error message
       crossfilter,
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'colors' does not exist on type 'Readonly... Remove this comment to see the full error message
       colors,
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'annoMatrix' does not exist on type 'Read... Remove this comment to see the full error message
       annoMatrix,
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'isUserAnno' does not exist on type 'Read... Remove this comment to see the full error message
-      isUserAnno,
+      isCellGuideCxg,
     } = this.props;
 
+    const { colorAccessor } = colors;
```

Comment:
not sure why this was changed to get from `colors` instead of keeping it in `{ colorAccessor } = asyncProps` below. But this seems to be causing bug #1022

Reply:
This is the PR against feature branch that added this code

https://github.com/chanzuckerberg/single-cell-explorer/pull/964/files